app.rb

```diff
@@ -0,0 +1,116 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  def initialize(menu)
+    @menu = menu
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No added book!'
+    else
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+    @menu.show_menu
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'No added person!'
+    else
+      @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    end
+    @menu.show_menu
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
```

Comment:
[OPTIONAL]

- I was able to create a student when I passed an alphabet as the age. It will be nice if you look into this bug 😄